### DISCUSSION TITLE
How about creating a client for another language (Go, Kotlin)

### DISCUSSION THREAD

**Discussion by moshe-kabala** [prisma/prisma#2027] [Q&A] [answered]
I've been following Prism for a while now and was very happy to see beta release for version 2

I have two questions about the future of the project

1.
The idea of ​​one source of truth is so plausible. However, not all micro services are written in the same language (JavaScript / TS) for a variety of reasons. Will you support more language in the future?

2.
Some models are sometime pass in streaming. how about genreated Protocol buffer schma from Prisma schema (or anther way to serialize/ deserialize model from bytes ) to continue the idea of ​​one source of truth to the model

Thanks. I really like the Prisma approach

**Comment by Sytten** [comment, marked as answer]
There is an experimental go client already: https://github.com/prisma/prisma-client-go

**Reply by moshe-kabala** [reply]
Thank you very much for seeing this (in the official header written for Javascript and Typescript) so hopefully it still continues.

**Reply by steebchen** [reply]
@moshe-kabala We'll indeed continue to work on the Go client!

**Comment by mCzolko** [comment]
Kotlin client would be amazing!

**Reply by janpio** [reply]
That would require a feature request issue then optimally so we can track the interest.

**Reply by mCzolko** [reply]
https://github.com/prisma/prisma/issues/11957 I made one. So let's see how it will do. Thank you

**Comment by sahalware** [comment]
In Kotlin we have this ORM library called Exposed which is backed by Jetbrains and I believe we don't really need Prisma there.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
